js/level1.js

```diff
@@ -4,34 +4,35 @@
     Comments
     ========
 
-    Let's start with comments. Comments are notes that people can read and
-    computers will ignore.
+    Let's start with comments. This is a comment. Comments are notes that people
+    can read and computers will ignore.
 
     They will help us to guide you through the JavaScript introduction
     journey.
 */
 
+// Single line comments look like this.
+
 /*
     Multi-line comments look like this.
+    When you are writing actual code, put it outside these comment blocks,
```

Comment:
People often get confused and put code inside the comments. Hopefully this makes it clearer